web_development_101/project_html_css.md

```diff
@@ -1242,6 +1242,8 @@ There are a couple of ways to go about doing this, but the simplest is this:
 * [AlfredaKava's Solution](https://github.com/alfredakava/google-homepage) | [View in browser (Easy)](https://cdn.rawgit.com/alfredakava/google-homepage/053498c9/index.html)
 * [Sinchana Gowda's Solution](https://github.com/sinchanagowda98/The_Odin_Project) | [View in browser](https://sinchanagowda98.github.io/The_Odin_Project/)
 * [Tanmayaudupa's Solution](https://github.com/tanmayaudupa/google-homepage) | [View in browser](https://tanmayaudupa.github.io/google-homepage/)
+* [Meghana Solution](https://github.com/meghana-vyakaranam/The_Odin_Project) | [View in browser](https://meghana-vyakaranam.github.io/The_Odin_Project/)
```

Comment:
@meghana-vyakaranam the view in broswer link doesn't work. You might need to re-arrange the files in your repository or look at using rawgit. Thanks, Rhys.